<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageSquarer</class>
 <widget class="QMainWindow" name="ImageSquarer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Squarer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: rgb(60, 60, 60);
color: rgb(255, 255, 255);
border-color: rgb(0, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushSkip">
        <property name="text">
         <string>Skip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushCrop">
        <property name="text">
         <string>Crop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
